src/Consumer/AbstractMailerConsumer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace AppBundle\Consumer;
+
+use AppBundle\Mailer\ClientInterface;
+use AppBundle\Mailer\Model\EmailRepository;
+use GuzzleHttp\Exception\ConnectException;
+use Symfony\Component\Validator\Constraints as Assert;
+
+abstract class AbstractMailerConsumer extends AbstractConsumer
+{
+    const CLIENT_ID = 'to_override';
```

Comment:
Overriding constants looks weird to me. If the value are "variable", then they should be defined as attributes.

Reply:
i did not change the behavior. but i agree, let's change this

Reply:
what about `abstract protected function getClientId(): string;` so that we ensure i'ts a non-nullable string? no property needed then?